pages/ecosystem/celestia.mdx

```diff
@@ -1 +1,70 @@
-# Celestia
\ No newline at end of file
+import { Steps } from "nextra-theme-docs";
+
+# Celestia
+[Celestia](https://celestia.org/) is a modular data availability network that makes it easy for anyone to securely launch their own blockchain.
+
+Celestia acts as a separate layer dedicated to storing and verifying transaction data, enabling app chains to focus solely on transaction execution and settlement. 
+This design facilitates significant scalability improvements in the blockchain ecosystem.
+
+## Key Features
+- Data Availability Sampling: Celestia allows light nodes on its network to efficiently verify data availability without downloading the entire dataset, this ensures the scalability of the network.
+- Modular Design: Celestia's modular design separates data availability from consensus and execution functionalities. This allows developers to build custom blockchains on top of Celestia, fostering innovation and flexibility within the ecosystem.
+- Proof-of-Stake: Celestia leverages a Proof-of-Stake consensus mechanism to guarantee the security and integrity of the data layer.
```

Comment:
while PoS is talked a lot as a consensus algorithm, it's an anti sybil mechanism. Tendermint is the consensus used by Celestia

Reply:
Okay, thanks :)